README.md

```diff
@@ -68,7 +68,26 @@ cat exported.json | jq -r '.meshes | to_entries[] | .value | to_entries[] | "\(.
 
 ## To Install
 
-We currently support `Blender 2.80`
+### Install Blender
+We currently support `Blender 2.80` The Blender command line executable should be installed and availble in your shell path. On MacOs you can install through homebrew to acheive this.
+
+```
```

Comment:
Can remove this block and instead:

To verify that Blender is in your path, run `blender --version` in your shell.